src/main/java/com/reimbursements/models/Authority.java

```diff
@@ -0,0 +1,91 @@
+package com.reimbursements.models;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+
+@Entity
+@Table(name = "authorities")
+public class Authority {
+	
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private int authorityID;
+	@Column(nullable=false)
+	private String authorityName;
+	@Column(nullable = false)
+	private int userID;
+	
+	public Authority() {
+		super();

```

Comment:
You don't need `super()`, because your class inherits from Object, the `super` is called either way.